embedded/embedded-test/src/test/java/org/mule/runtime/module/embedded/EmbeddedContainerTestCase.java

```diff
@@ -218,4 +221,9 @@ private void doWithinApplication(String applicaitonFolder, Consumer<Integer> por
   private URI getClasspathResourceAsUri(String resource) throws URISyntaxException {
     return getClass().getClassLoader().getResource(resource).toURI();
   }
+
+  @Override
+  public int getTestTimeoutSecs() {
+    return 20 * super.getTestTimeoutSecs();
```

Comment:
this test was currently downloading all of its dependencies from a remote repo. It took 11 minutes on my computer to run.